### PR TITLE
Rewrite toolchain configuration in preference order

### PR DESCRIPTION
**Pull request by theotherjimmy**
And add a section about the priority between multiple toolcahin
configurations

### CODE REVIEW

File: docs/tools/offline/cli.md [hunk 1]
```diff
@@ -74,51 +74,61 @@ Mbed CLI requires adding the following to the system `PATH`:
...
 ```
 $ mbed config -G ARM_PATH "C:\Program Files\ARM"
 [mbed] C:\Program Files\ARM now set as global ARM_PATH
 ```
+The `-G` switch tells Mbed CLI to set this as a global setting, allowing all projects owned by this user to compile with Arm Compiler 5. You may instead set the `ARM_PATH` using a project local setting by omitting the `-G` command line switch.
+
+Mbed CLI supports a setting for each toolchain path. These setting are listed below, along with a description of what path is expected in each setting.
 
-The `-G` switch tells Mbed CLI to set this as a global setting, rather than local for the current program.
+* `ARM_PATH`: The path to the *base* directory of your Arm Compiler installation. This should be the directory containing the directory containing the binaries for `armcc` and friends.
```
Comment (AnotherButler) [inline docs/tools/offline/cli.md]: Query: The directory containing the binaries? Or the directory containing the directory containing the binaries?

Comment (theotherjimmy) [reply docs/tools/offline/cli.md]: For this one, if the binary is `<some-path>/arm/bin/armcc` then the path should be `<some-path>/arm`. Do you think an example would help here?

Comment (AnotherButler) [reply docs/tools/offline/cli.md]: Yes

**Comment by theotherjimmy** [comment]
@AnotherButler I added the examples as requested.

**Pull request merged**